src/db_impl.cc

```diff
@@ -496,6 +494,31 @@ Options TitanDBImpl::GetOptions(ColumnFamilyHandle* column_family) const {
   return options;
 }
 
+Status TitanDBImpl::SetOptions(
+    ColumnFamilyHandle* column_family,
+    const std::unordered_map<std::string, std::string>& new_options) {
+  auto opts = new_options;
```

Comment:
we should hold `mutex_` lock here I think, for accessing `titan_table_factory_`.